### PR TITLE
Improve Tikui generated style

### PR DESCRIPTION
**Pull request by Gnuk**

### CODE REVIEW

**Review by murdos — DISMISSED** [review]

File: src/main/java/tech/jhipster/lite/module/domain/file/JHipsterModuleFiles.java [hunk 1]
```diff
@@ -119,6 +119,12 @@ public JHipsterModuleFileBatchBuilder addTemplate(String file) {
       return add(source.template(file), destination.append(file));
     }
 
+    public JHipsterModuleFileBatchBuilder addTemplates(String... files) {
+      Arrays.stream(files).forEach(this::addTemplate);
```
Comment (DamnClin) [inline src/main/java/tech/jhipster/lite/module/domain/file/JHipsterModuleFiles.java:123]: `Arrays.stream` => `Stream.of` (more aggressive to null values)

File: src/main/resources/config/application.yml [hunk 2]
```diff
@@ -68,7 +68,6 @@ application:
 jhlite-hidden-resources:
   slugs:
     - svelte-core
-    - tikui
```
Comment (DamnClin) [inline src/main/resources/config/application.yml:71]: need to add it in all front applications in `generate.sh`

File: src/main/resources/generator/client/tikui/style/template/template-page/_template-page.scss.mustache [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  &--content {
+    box-sizing: border-box;
+    margin: 0 auto;
+    background-color: var(--color-background);
+    padding: var(--spacing-l);
+    width: 100%;
+    max-width: var(--content-width);
+    min-height: 100vh;
+
+    @media screen and (width >= 1200px) {
```
[the remark this answers is not in the document]
Comment (Gnuk) [reply src/main/resources/generator/client/tikui/style/template/template-page/_template-page.scss.mustache:16]: Yes but CSS doesn't allow this, so I can just add a CSS variable if you want

File: src/main/java/tech/jhipster/lite/module/domain/file/JHipsterModuleFiles.java [hunk 4]
```diff
@@ -119,6 +119,12 @@ public JHipsterModuleFileBatchBuilder addTemplate(String file) {
       return add(source.template(file), destination.append(file));
     }
 
+    public JHipsterModuleFileBatchBuilder addTemplates(Collection<String> files) {
```
Comment (DamnClin) [inline src/main/java/tech/jhipster/lite/module/domain/file/JHipsterModuleFiles.java:122]: Assert.notNull("files", files)

**Review by DamnClin — DISMISSED** [review]
LGTM, @pascalgrimaud I think this deserve a Bounty. WDYT?

**Comment by pascalgrimaud** [comment]
thanks a lot for your great work here @Gnuk !!

**Comment by Gnuk** [comment]
Thank you @pascalgrimaud and @DamnClin 

Claimed at https://opencollective.com/generator-jhipster/expenses/223975

**Comment by pascalgrimaud** [comment]
@Gnuk : approved

**Pull request merged**